src/js/post-911-gib-status/components/EnrollmentPeriod.jsx

```diff
@@ -1,35 +1,99 @@
+import drop from 'lodash/fp/drop';
+
 import React from 'react';
 import PropTypes from 'prop-types';
+import Scroll from 'react-scroll';
 
 import InfoPair from './InfoPair';
 
+import { getScrollOptions } from '../../common/utils/helpers';
 import { formatDateShort } from '../utils/helpers';
 
+const scroller = Scroll.scroller;
+
 class EnrollmentPeriod extends React.Component {
+  constructor() {
+    super();
+    this.scrollToEnrollment = this.scrollToEnrollment.bind(this);
+    this.toggleHistory = this.toggleHistory.bind(this);
+    this.state = { historyOpen: false };
+  }
+
+  scrollToEnrollment() {
+    const options = getScrollOptions({ delay: 2 });
+    scroller.scrollTo(`collapsible-${this.props.id}`, options);
+  }
+
+  toggleHistory() {
+    const willOpen = !this.state.historyOpen;
+    this.setState({ historyOpen: !this.state.historyOpen });
+    if (willOpen) {
+      this.scrollToEnrollment();
+    }
+  }
+
   render() {
-    const { enrollment } = this.props;
+    const { enrollment, id } = this.props;
+    const amendments = enrollment.amendments || [];
+    const firstAmendment = amendments.length > 0 && amendments[0];
+
+    // TODO: should fullTimeHours be used anywhere?
+    // const fullTimeHours = <InfoPair label="Full-time Hours" value={enrollment.fullTimeHours}/>;
+    const onCampusHours = <InfoPair label="On-campus Hours" value={enrollment.onCampusHours}/>;
+    const onlineHours = <InfoPair label="Online Hours" value={enrollment.onlineHours}/>;
+    const typeOfChange = firstAmendment &&
```

Comment:
I am beginning to think that these 2 things don't belong here because
  * the first amendment has its own `onCampusHours` and `onlineHours` values which are being ignored here. Unless the first amendment overrides the original enrollment's hours? Design & product question so I'll add it to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3250